packages/core/src/sass/app/_common/layout/footer.scss

```diff
@@ -6,7 +6,7 @@
     width: 100%;
     height: $FOOTER_HEIGHT;
     box-sizing: border-box;
-    z-index: 3;
+    z-index: 4;
```

Comment:
The run panel extends over the footer which is incorrect, instead of increasing the z-index, we should limit the height of the run panel animation container.

Reply:
Dear Aaron I changed the size of 'run panel' but I believe that it's better if z-index also should be 4 because our entire footer panel should be over the run panel

Reply:
We shouldn't touch the core package when it's not needed, it's better if our app limits itself. Could you revert this?